lib/backend/syncersv1/updateprocessors/rules_test.go

```diff
@@ -36,30 +36,30 @@ var _ = Describe("Test the Rules Conversion Functions", func() {
 				Code: &incode,
 			},
 			Source: apiv3.EntityRule{
-				Nets:        []string{"10.100.10.1"},
+				Nets:        []string{"10.100.10.1/32"},
```

Comment:
Are we no longer allowing IP addresses in here?

Reply:
correct.